android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsAppOpenModule.kt

```diff
@@ -56,6 +56,7 @@ class ReactNativeGoogleMobileAdsAppOpenModule(reactContext: ReactApplicationCont
       activity,
       adUnitId,
       adRequest,
+      AppOpenAd.APP_OPEN_AD_ORIENTATION_PORTRAIT,
```

Comment:
Passing an orientation is deprecated: https://developers.google.com/android/reference/com/google/android/gms/ads/appopen/AppOpenAd#load(android.content.Context,%20java.lang.String,%20com.google.android.gms.ads.AdRequest,%20com.google.android.gms.ads.appopen.AppOpenAd.AppOpenAdLoadCallback

Reply:
Thanks for looking into this @dylancom and apologies for the delay in response. gradle downgraded the android sdk in my setup:

```
+--- project :react-native-google-mobile-ads
|    +--- com.google.android.ump:user-messaging-platform:2.0.0 (*)
|    +--- com.facebook.react:react-native:+ -> 0.69.6-3 (*)
|    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.21 (*)
|    \--- com.google.android.gms:play-services-ads:22.0.0 -> 21.1.0 (*)
Overriding dependency com.google.android.gms:play-services-ads version 22.0.0 --> 21.1.0
```

At least that's why I ran into this issue. I'm not sure why it's downgraded and I'll keep digging there. Thanks again for all that you do.